interface.html

```diff
@@ -24,10 +24,10 @@
           </div>
         </div>
         <div class="helper-holder clearfix">
-          <div class="alert alert-danger error">The link doesn’t contain a video thumbnail or it is not supported. Please try again later if the video is recently updated.</div>
-          <div class="alert alert-warning warning">The link doesn't support inline video playback. Your video will play in the in-app browser (for native apps) or a new tab (for web apps).</div>
-          <a id="try-stream-single" href="#">Show me an example</a>
-          <a href="#" class="pull-right hidden" data-refresh>Refresh thumbnail</a>
+          <a id="try-stream-single" href="#" {{#if url}}class="invisible"{{/if}}>Show me an example</a>
```

Comment:
why "invisible" instead of the more common "hidden"

Reply:
`hidden` collapses the container and ends up moving the elements underneath. `invisible` just makes it invisible through `visibility: hidden` without changing the layout.